src/main/java/ch/njol/skript/registrations/Feature.java

```diff
@@ -11,20 +13,154 @@
  * Experimental feature toggles as provided by Skript itself.
  */
 public enum Feature implements Experiment {
-	EXAMPLES("examples", LifeCycle.STABLE),
-	QUEUES("queues", LifeCycle.EXPERIMENTAL),
-	FOR_EACH_LOOPS("for loop", LifeCycle.EXPERIMENTAL, "for [each] loop[s]"),
-	SCRIPT_REFLECTION("reflection", LifeCycle.EXPERIMENTAL, "[script] reflection"),
-	CATCH_ERRORS("catch runtime errors", LifeCycle.EXPERIMENTAL, "error catching [section]"),
-	TYPE_HINTS("type hints", LifeCycle.EXPERIMENTAL, "[local variable] type hints"),
-	DAMAGE_SOURCE("damage source", LifeCycle.EXPERIMENTAL, "damage source[s]")
+
+	EXAMPLES("Examples",
+		"""
+			A section used to provide examples inside code.
+			
+			```
+			example:
+				kick the player due to "you are not allowed here!"
+			```
+			""",
```

Comment:
```suggestion
		A section used to provide examples inside code.
		
		```
		example:
			kick the player due to "you are not allowed here!"
		```
		""",
```
keeping it on the same line makes it easier to read